trusona.gemspec

```diff
@@ -37,6 +38,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'guard-yard', '~> 2.2'
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
+  spec.add_development_dependency 'rspec-wait'
```

Comment:
`WARNING:  open-ended dependency on rspec-wait (>= 0, development) is not recommended
  use a bounded requirement, such as '~> x.y'`